src/components/Checkbox.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {ForwardedRef, forwardRef, KeyboardEvent as ReactKeyboardEvent} from 'react';
+import {GestureResponderEvent, StyleProp, View, ViewStyle} from 'react-native';
+import styles from '@styles/styles';
+import * as StyleUtils from '@styles/StyleUtils';
+import themeColors from '@styles/themes/default';
+import CONST from '@src/CONST';
+import ChildrenProps from '@src/types/utils/ChildrenProps';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import PressableWithFeedback from './Pressable/PressableWithFeedback';
+
+type CheckboxProps = ChildrenProps & {
+    /** Whether checkbox is checked */
+    isChecked?: boolean;
+
+    /** A function that is called when the box/label is pressed */
+    onPress: () => void;
+
+    /** Should the input be styled for errors  */
+    hasError?: boolean;
+
+    /** Should the input be disabled  */
+    disabled?: boolean;
+
+    /** Additional styles to add to checkbox button */
+    style?: StyleProp<ViewStyle>;
+
+    /** Additional styles to add to checkbox container */
+    containerStyle?: StyleProp<ViewStyle>;
+
+    /** Callback that is called when mousedown is triggered. */
+    onMouseDown?: () => void;
+
+    /** The size of the checkbox container */
+    containerSize?: number;
+
+    /** The border radius of the checkbox container */
+    containerBorderRadius?: number;
+
+    /** The size of the caret (checkmark) */
+    caretSize?: number;
+
+    /** An accessibility label for the checkbox */
+    accessibilityLabel: string;
+};
+
+function Checkbox(
+    {
+        isChecked = false,
+        hasError = false,
+        disabled = false,
+        style = [],
+        containerStyle = [],
```

Comment:
```suggestion
        style,
        containerStyle,
```
Default value not necessary.

Reply:
Removed!